test/puppeteer.test.js

```diff
@@ -204,51 +206,36 @@ describe('Apify.launchPuppeteer()', () => {
         }
     });
 
-    test('should allow to use Apify proxy', async () => {
+    test('should allow to use proxyConfiguration', async () => {
         process.env[ENV_VARS.PROXY_PASSWORD] = 'abc123';
         process.env[ENV_VARS.PROXY_HOSTNAME] = 'my.host.com';
         process.env[ENV_VARS.PROXY_PORT] = 123;
+        const status = { connected: true };
+        const fakeCall = async () => {
+            return { body: status };
+        };
 
-        const mock = sinon.mock(actor);
-        mock.expects('getApifyProxyUrl')
-            .once()
-            .withArgs({
-                session: 'xxx',
-                groups: ['yyy'],
-                groupsParamName: 'options.apifyProxyGroups',
-                sessionParamName: 'options.apifyProxySession',
-            })
-            .returns(null); // Return null so that it doesn't start proxy-chain
+        const stub = sinon.stub(utilsRequest, 'requestAsBrowser').callsFake(fakeCall);
+        const proxyConfiguration = await Apify.createProxyConfiguration({
+            groups: ['GROUPyyy'],
+        });
 
         try {
             await Apify
                 .launchPuppeteer({
-                    useApifyProxy: true,
-                    apifyProxySession: 'xxx',
-                    apifyProxyGroups: ['yyy'],
+                    proxyConfiguration,
```

Comment:
This shouldn't work. The test should fail, but it doesn't. That means it's not actually testing anything.

Reply:
You are right, it is not necessary to test the `proxyConfiguration` here. Only the `proxyUrl` comes to `launchPuppeteer()` and it is already tested.